src/cratedb_about/cli.py

```diff
@@ -28,16 +28,31 @@ def cli(ctx: click.Context) -> None:
     type=str,
     required=False,
     default=None,
-    help="URL of the outline file. By default, the builtin outline is used.",
+    help="URL to the outline file. By default, the built-in outline is used.",
 )
 @click.option(
-    "--format", "-f", "format_", type=click.Choice(["markdown", "yaml", "json"]), default="markdown"
+    "--format",
+    "-f",
+    "format_",
+    type=click.Choice(["llms-txt", "markdown", "yaml", "json"]),
+    default="markdown",
+    help="Output format",
 )
-def outline(url: str, format_: t.Literal["markdown", "yaml", "json"] = "markdown") -> None:
+@click.option(
+    "--optional",
+    "-o",
```

Comment:
I wonder if `-o` can viewed as an alias `--output`. Maybe `-O` is a better alias for `--optional`?

Reply:
Good point, thanks. I've removed the alias completely per 939b770b9e.